questions/qa-ltr-scripts-in-rtl.zh-hans.html

```diff
@@ -0,0 +1,425 @@
+<link rel="stylesheet" href="../style/article-2022.css">
+<link rel="stylesheet" href="qa-ltr-scripts-in-rtl-data/local.css">
+<link rel="copyright" href="#copyright">
+<script src="../javascript/prism.js"></script>
+<link rel="stylesheet" href="../style/prism.css">
+</head>
+
+<body>
+<header>
+<nav id="mainNavigation"></nav><script>document.getElementById('mainNavigation').innerHTML = mainNavigation</script>
+
+<h1>从右向左显示从左向右书写的文字</h1>
+</header>
+
+
+<div id="audience">
+<div id="updateInfo"></div><script>document.getElementById('updateInfo').innerHTML = g.updated</script>
+</div>
+
+<p>除了常见的由右至左书写的书写系统（阿拉伯字母、希伯来字母、西非书面字母、叙利亚字母、它拿字母等等）之外，还有许多其他由左至右（left-to-right, LTR）书写的书写系统曾经由右至左（right-to-left, RTL）书写，包括中文、日文、圣书体、提非纳文字、古斯堪地纳维亚语以及许多其他古代文字。</p>
+
+<div class="figWrap">
+<figure id="japanese_stamp">
+<img src="qa-ltr-scripts-in-rtl-data/japanese_stamp.jpg" alt="日本邮票的图片" style="width: 300px;">
+<figcaption>日本第一张纪念邮票，纪念1894年天皇结婚25周年，上方的横幅由右至左书写。</figcaption>
+</figure>
+</div>
+<p>由于在Unicode标准中，这些文字是强LTR方向的，我们不能简单地使用Unicode双向文本算法来正确显示它们。此外，其中一些文字在由右至左书写时会对字形进行镜像。</p>
+<p>本文提出了使用HTML和CSS为这些语言生成从右到左的文本的方法。</p>
+
+
+
+<section id="use_cases">
+<h2>用例</h2>
+
+<p>实际上的用例很少。你可能偶尔会遇到由右至左书写的台湾或图阿雷格语文字，但大多数用例都是为了重现古代用法。</p>
+
+<p>古代文字，例如古埃及圣书体，在现代通常由左至右书写。然而，最初的铭文或书面资料通常是由右至左书写的。就埃及而言，纪念碑上圣书字的方向是根据整体的美观来决定的。在一个人的画像旁边你还会看到圣书字从两边向中间的人脸水平排列。不仅如此，字形也会被镜像，这样你也可以朝着任何字形的正面或开头阅读。</p>
+
+<p>因此，现代埃及古物学者写作</p>
+<p style="text-align: center;"><img src="qa-ltr-scripts-in-rtl-data/egyptian-ltr.png" alt="𓅓𓂞𓎡𓏏𓏦𓀭"></p>
+<p>的序列在原版中从右到左排序，而且字形是被镜像的。</p>
+
+<div class="figWrap">
+<figure id="hieroglyphs">
+<img src="qa-ltr-scripts-in-rtl-data/egyptian.jpeg" alt="Excerpt from ." style="width: 60%;">
+<figcaption>埃及石灰石雕刻的细节（BM EA 585）</figcaption>
+</figure>
+</div>
+<p>虽然你可能很少需要这样做，但如果你想的话，我们将在这里介绍如何让文本看起来像原始文本。</p>
+
+
+
+<p>就中文和日文而言，你会遇到的大多数现代文本片段都很短，并且通常设计为古风，例如为了让游客回想古代中国和日本。</p>
+
+<p>对于日语，sci.lang.japan新闻组上的一篇<a href="https://www.sljfaq.org/afaq/right-to-left.html">有趣的文章</a>解释说，在明治时期之前，除非有空间限制，文字通常是直排的。19世纪末，横排开始作为直排的严肃替代品，引发了对行内文字书写方向的初步实验。</p>
+<blockquote>
+<p>明治时代（1868-1912），在横书流行之初，出现了一种短暂的书写方式，称为“右横书”，与现代的书写方式——左横书相反。这和阿拉伯语、希伯来语等语言的由右至左横向书写风格类似，在页面左侧换行……这种形式从未被广泛使用，也没有保留下来。</p></blockquote>
+
+<p>下图是2000年左右的一份台湾的报纸，这种情况现已很少见。我们可以看到，中间偏右横向书写的标题是从右到左书写的，和与标题下方垂直行的行进方向相同，但它还包括从左到右书写的文本“8.6%”。另请注意，左边图片下方标题中的文本包含多行（而且<strong>未与网格对齐</strong>），也是由右至左书写的。</p>
+
+<div class="figWrap">
+<figure id="taiwanese_newspaper">
+<img src="qa-ltr-scripts-in-rtl-data/taiwanese_newspaper.jpg" alt="一份台湾报纸" style="width: 100%;">
+<figcaption>一份台湾报纸的一部分，其中的标题和图片说明由右向左横向书写</figcaption>
+</figure>
+</div>
+
+
+
+<section id="use_case_types">
+<h3>两种不同类型的用例</h3>
+
+<p>上面的用例可以分为两类，而我们下面提出的两种解决方案可以满足这两类用例。</p>
+<p><strong>第一种用例，可能也是最有可能出现的一种，是表示说明性文本的简短示例。</strong>在这种情况下，如果字符不以相反的顺序显示，则这个例子将失去其意义。例如，如果你得到了错误的文字排列顺序：“文本<span lang="ber">ⴰⵙⵉⴹⵢⴰⵙ⵰</span>最初是这样写的：<bdo lang="ber" dir="ltr">ⴰⵙⵉⴹⵢⴰⵙ⵰</bdo>”，读者会摸不着头脑。因此，在这种情况下，我们希望确保读者看到的<strong>始终是反向排列的文字</strong>，而我们<strong>通过标记语言</strong>来做到这一点。</p>
```

Comment:
The strong text in these two paras doesn't stand out much.  Do you have any idea about what we should add to the chinese-specific styling to create a more visible distinction?

Reply:
I think the following style will make it stand out:

```css
font-weight: 900;
color: black;
```

And it should be in our global CSS for `zh`, because other Chinese articles and pages also have this issue.

Reply:
Thanks. Fixed now.